tests/data/mini_c_test/includes/a.h

```diff
@@ -7,6 +7,11 @@ void a_step(void);
 extern int a_u1;
 extern int a_y1;
 
```

Comment:
`s_t` is introduced as type but is not used in any variable declaration. 

I suggest to add structs as simple variable, return type of a function and argument of a function, maybe also as a pointer (?) to the test data and add test(s) on it.

Reply:
Right, it was intended for a test [here](https://github.com/avengineers/hammocking/blob/d488a3d067134da08d871a219e885504faf4b14a/tests/hammocking_test.py#L256) but that didn't work.
We parse the variables by snippets, so the compiler does not see the include file when it encounters the variable
(that's why all types suddenly turned to `int`).
Should enhance the test here. It should more focus on the aspects of the functions for the template; the writer interfaces are better tested [here](https://github.com/avengineers/hammocking/blob/d488a3d067134da08d871a219e885504faf4b14a/tests/hammocking_test.py#L63).

Reply:
I see. That‘s reasonable.

From my side that‘s all good to go.